bom/runtime/pom.xml

```diff
@@ -2921,6 +2862,23 @@
                 <artifactId>google-http-client</artifactId>
                 <version>${google-http-client.version}</version>
             </dependency>
+
+            <!-- Kogito -->
+            <dependency>
+                <groupId>org.kie.kogito</groupId>
+                <artifactId>kogito-api</artifactId>
+                <version>${kogito.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.kie.kogito</groupId>
+                <artifactId>kogito-drools</artifactId>
+                <version>${kogito.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.kie.kogito</groupId>
+                <artifactId>drools-core-static</artifactId>
+                <version>${kogito.version}</version>
+            </dependency>
```

Comment:
Why this is still needed if kogito is being removed from the Quarkus codebase?

Reply:
@gastaldi That is required by optaplanner extension.